### DISCUSSION TITLE
No WALC-Tools if using Whatsapp-Business

### DISCUSSION THREAD

**Discussion by fleixi** [cstayyab/WALC#247] [Troubleshooting]
I am using this client with the Whatsapp-Business-App connected to my Phone. The business-tools working just fine but there is no button for the walc-tools. Or do i miss them somehow?

**Comment by daffamumtaz2361** [comment]
I never tested the app with WhatsApp business account. Could you provide some screenshots of how it looks?

**Comment by fleixi** [comment]
![Screenshot_20240405_133325](https://github.com/WAClient/WALC/assets/7696873/7fc8e004-7eac-49c1-bca3-70d7b4a5de03)

It is in German, i hope this is no problem. im not 100% sure but i think you could install and use the business-app without have to register any spezial business account by whatsapp.

If you need some more screenshots just ask and point me to the area to shot.

**Comment by daffamumtaz2361** [comment]
I thought you meant you're using business account. How do you install this business app? Is it a script? How are you using it together with WALC?

**Reply by fleixi** [reply]
Its in the Playstore "WhatsApp Business" https://play.google.com/store/apps/details?id=com.whatsapp.w4b&hl=de&gl=US

I have started WALC and scanned the QR-Code. Nothing special. Just this app and the walc from the arch-linux aur
